modules/search/pages/default-token-filters-reference.adoc

```diff
@@ -0,0 +1,213 @@
+= Default Token Filters
+:page-topic-type: reference
+:description: Use a token filter to filter a tokenizer's results and get better search result matches. 
+
+[abstract]
+{description}
+
+The Search Service's xref:customize-index.adoc#token-filters[token filters] work with xref:customize-index.adoc#tokenizers[tokenizers] to filter search input tokens. 
+Tokens can come from the content of your Search index or a Search query.
+
+For more information about token filters, see xref:customize-index.adoc#token-filters[Token Filters].
+
+The following token filters are available: 
+
+[cols="1,2"]
+|====
+|Token Filter Type |Description 
+
+|apostrophe | Removes all characters after an apostrophe (') from tokenizer results. Also removes the apostrophe.
```

Comment:
That single `'` has confused github's syntax highlighter (e.g. rest of this page is italicised as it thinks it's part of the same string... 🤦 
Please double-check that the actual Asciidoc output looks fine?

Reply:
Verified many times, never noticed an issue.